ansible/setup-convert2rhel.yml

```diff
@@ -26,12 +26,6 @@
         msg: "Distribution must be CentOS or Oracle Linux"
       when: ansible_distribution != 'CentOS' and ansible_distribution != 'OracleLinux'
 
-    - name: Validate Release Version (6.10, 7.9, 8.5)
```

Comment:
I've removed this check as it's difficult to maintain (we forgot to update it many times) and it's not critical.